### PR TITLE
first milestone portfolio page

### PR DESCRIPTION
**Pull request by frnega**
i tried to do template for by adapting to my name

### CODE REVIEW

**Review by zainsadaqat — CHANGES_REQUESTED** [review]
# Required Changes 🔁
# Hello @frnega  😄 

You're doing a great job on your project. We're proud of you. Good job so far! 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
  ✅  Correct use of Github Flow
  ✅  No linter errors
  ✅  Descriptive Pull Request

## Required Changes ♻️
[FIGMA DESIGN]
Screenshot 2022-08-19 at 10 14 01 PM


[YOUR DESIGN]
Screenshot 2022-08-19 at 10 13 27 PM

- [ ] Hamburger icon is missing. 👍🏻 

- [ ] Please use the required font family and color as in the Figma design 👍🏻 

- [ ] For semantic reasons and to make it SEO friendly Please wrap the navigation bar (logo + hamburger icon)with the `<nav></nav/>` tag 👍🏻 

- [ ] For semantic reasons and to make it SEO friendly Please wrap the `<nav></nav>` with `<header>` tag 😀 

- [ ] Please use paragraph tag `p` for the description instead of `h3` tag. 😃 


_Check the comments under the review._




## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by frnega** [comment]
changed the header tag to paragraph
the hamburger is next to frehiywot logo
inserted the hamburger and logo inside navbar  and also inside head tag

File: index.html [hunk 1]
```diff
@@ -5,6 +5,7 @@
 		<meta charset="utf-8">
 		<link rel="stylesheet" href="style.css">
 		<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+		<nav>
```
Comment (Ugboaja-Uchechi) [inline index.html:8]: - [ ] Kindly move your `nav` tag and everything in it to the `body` tag 🙏 
- [ ] Your `nav` tag should be wrapped in a `header` tag like the previous code reviewer mentioned E.g 
`<header>`  `<nav>`  `</nav>` `</header>`

File: style.css [hunk 2]
```diff
@@ -16,7 +16,7 @@ html {
 #nameme {
   width: 81px;
   height: 16px;
-  font-family: 'Times New Roman', Times, serif;
+  font-family: 'Roboto', Times, serif;
```
Comment (Ugboaja-Uchechi) [inline style.css:19]: - [ ] Kindly make sure your logo and menu are spaced properly and looks as close as possible to the Figma design 🙏 

Figma Design | Your Design
------------- | -------------
image  |  image

File: style.css [hunk 3]
```diff
@@ -61,7 +61,7 @@ html {
 .headline {
   width: 327px;
   height: 55px;
-  font-family: 'Times New Roman', Times, serif;
+  font-family: 'Roboto', Times, serif;
```
Comment (Ugboaja-Uchechi) [inline style.css:64]: - [ ] Good job specifying the font family👍. Kindly make sure you import the font family from [google fonts](https://fonts.google.com/specimen/Roboto) like the previous reviewer mentioned🙏 .

File: style.css [hunk 4]
```diff
@@ -91,7 +91,7 @@ html {
   flex-grow: 0;
 }
 
-.headline3 {
+p {
   width: 327px;
```
Comment (Ugboaja-Uchechi) [inline style.css:95]: - [ ] Like the previous code reviewer mentioned please make sure your design looks as close as possible to the Figma design 🙏 
- [ ] Kindly make sure your site has the right background image 🙏 

Figma Design | Your Design
------------- | -------------
image  |  image

File: style.css [hunk 5]
```diff
@@ -77,7 +77,7 @@ html {
 .headline2 {
   width: 327px;
   height: 87px;
-  font-family: 'Times New Roman', Times, serif;
+  font-family: 'Roboto', Times, serif;
   font-style: normal;
```
Comment (Ugboaja-Uchechi) [inline style.css:81]: - [ ] [OPTIONAL] Please make you use `flexbox`  🙏 . This gives us a lot of options to align all our elements, plus, it's responsive

**Review by Ugboaja-Uchechi — CHANGES_REQUESTED** [review]
Hi @frnega ,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Linters are passing ✔️ 
- Good commit messages 👍 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [ ] KIndly consider committing your code more this would help you keep track of your code better 🙏 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**